There is one failed build test:

```
error: builder for '/nix/store/4826npgqcfyfzzma6zn5s3k6n0acri3p-gitlab-container-registry-3.77.0.drv' failed with exit code 1;
       last 10 log lines:
       > time="2023-06-19T10:19:08Z" level=info msg="deleting manifest metadata in batches" batch_count=1 batch_max_size=100 go_version=go1.20.5 version=unknown
       > time="2023-06-19T10:19:08Z" level=info msg="preparing batch" batch_number=1 batch_total=1 go_version=go1.20.5 version=unknown
       > time="2023-06-19T10:19:08Z" level=info msg="deleting batch" batch_number=1 go_version=go1.20.5 manifests=2 tags=2 total=4 version=unknown
       > time="2023-06-19T10:19:08Z" level=info msg="batch deleted" batch_number=1 count=4 duration_s=2.0248e-05 go_version=go1.20.5 version=unknown
       > time="2023-06-19T10:19:08Z" level=info msg="manifest metadata deleted" duration_s=5.7226e-05 go_version=go1.20.5 version=unknown
       > OOPS: 44 passed, 1 skipped, 1 FAILED
       > --- FAIL: Test (84.72s)
       > FAIL
       > FAIL   github.com/docker/distribution/registry/storage/driver/inmemory 92.565s
       > FAIL
       For full logs, run 'nix log /nix/store/4826npgqcfyfzzma6zn5s3k6n0acri3p-gitlab-container-registry-3.77.0.drv'.
```

--- a/pkgs/applications/version-management/gitlab/gitlab-container-registry/Disable-inmemory-storage-driver-test.patch
+++ b/pkgs/applications/version-management/gitlab/gitlab-container-registry/Disable-inmemory-storage-driver-test.patch
@@ -1,0 +1,38 @@
+From bc359e8f51a17ba759121339e87e90eed16e98fe Mon Sep 17 00:00:00 2001
+From: Yaya <mak@nyantec.com>
+Date: Tue, 20 Jun 2023 10:01:23 +0000
+Subject: [PATCH] Disable inmemory storage driver test
+
+---
+ .../storage/driver/inmemory/driver_test.go    | 19 -------------------
+ 1 file changed, 19 deletions(-)
+ delete mode 100644 registry/storage/driver/inmemory/driver_test.go
+
+diff --git a/registry/storage/driver/inmemory/driver_test.go b/registry/storage/driver/inmemory/driver_test.go
+deleted file mode 100644
+index dbc1916f..00000000
+--- a/registry/storage/driver/inmemory/driver_test.go
++++ /dev/null
+@@ -1,19 +0,0 @@
+-package inmemory
+-
+-import (
+-	"testing"
+-
+-	storagedriver "github.com/docker/distribution/registry/storage/driver"
+-	"github.com/docker/distribution/registry/storage/driver/testsuites"
+-	"gopkg.in/check.v1"
+-)
+-
+-// Hook up gocheck into the "go test" runner.
+-func Test(t *testing.T) { check.TestingT(t) }
+-
+-func init() {
+-	inmemoryDriverConstructor := func() (storagedriver.StorageDriver, error) {
+-		return New(), nil
+-	}
+-	testsuites.RegisterSuite(inmemoryDriverConstructor, testsuites.NeverSkip)
+-}
+-- 
+2.40.1
+

--- a/pkgs/applications/version-management/gitlab/gitlab-container-registry/default.nix
+++ b/pkgs/applications/version-management/gitlab/gitlab-container-registry/default.nix
@@ -14,6 +14,10 @@ buildGoModule rec {
 
   vendorHash = "sha256-/ITZBh0vRYHb6fDUZQNRwW2pmQulJlDZ8EbObUBtsz4=";
 
+  patches = [
+    ./Disable-inmemory-storage-driver-test.patch
+  ];
+
   postPatch = ''
     substituteInPlace health/checks/checks_test.go \
       --replace \

